huge-cache/src/main/scala/org/broadinstitute/dig/aggregator/methods/hugecache/HugeCacheStage.scala

```diff
@@ -0,0 +1,61 @@
+package org.broadinstitute.dig.aggregator.methods.hugecache
+
+import org.broadinstitute.dig.aggregator.core.{Context, Input, Outputs, Stage}
+import org.broadinstitute.dig.aws.emr.{ClusterDef, Job}
+
+/** This is a stage in your method.
+  *
+  * Stages take one or more inputs and generate one or more outputs. Each
+  * stage consists of a...
+  *
+  *   - list of input sources;
+  *   - rules mapping inputs to outputs;
+  *   - make function that returns a job used to produce a given output
+  *
+  * Optionally, a stage can also override...
+  *
+  *   - its name, which defaults to its class name
+  *   - the cluster definition used to provision EC2 instances
+  */
+class HugeCacheStage(implicit context: Context) extends Stage {
+
+  val variantCqsDir: String   = "out/varianteffect/cqs/"
+  val nearestGenesDir: String = "out/huge/nearestgenes/"
+  val cacheDir                = "out/huge/cache/"
+
+  val variantCqs: Input.Source   = Input.Source.Success(variantCqsDir)
+  val nearestGenes: Input.Source = Input.Source.Success(nearestGenesDir)
+
+  /** Source inputs. */
+  override val sources: Seq[Input.Source] = Seq(variantCqs, nearestGenes)
+
+  /* Define settings for the cluster to run the job.
+   */
+  override val cluster: ClusterDef = {
+    super.cluster.copy(instances = 10)
```

Comment:
Is 10 instances strictly necessary here? How long does it take to run with a more normal number like 5?